page/layouts/shortcodes/faq.html

```diff
@@ -0,0 +1,37 @@
+{{ $_hugo_config := `{ "version": 1 }` }}
+<div class="expand">
+    <div class="expand-label" style="cursor: pointer;" onclick="$h = $(this);$h.next('div').slideToggle(100,function () {$h.children('i').attr('class',function () {return $h.next('div').is(':visible') ? 'fas fa-chevron-down' : 'fas fa-chevron-right';});});">
+        <i style="font-size:x-small;" class="fas fa-chevron-right"></i>
+        <span>
+        {{$expandMessage := T "Expand-title"}}
+    	{{ if .IsNamedParams }}
+    	{{.Get "default" | default $expandMessage}}
+    	{{else}}
+    	{{.Get 0 | default $expandMessage}}
+    	{{end}}
+    	</span>
+    </div>
+    <div class="expand-content" style="display: none;">
+        {{.Inner | safeHTML}}
+    </div>
+</div>
```

Comment:
You might not need to duplicate the whole `expand` template here. Have you tried if it works with

```
{{ partial "expand.html" . }}
```

? (or `"layouts/shortcodes/expand.html"`)

Reply:
could not solve this.
seams like `partial` does not work with other shortcodes, but only with partials.

Maybe .parent could help out here ... but could not find a working solution eighter.
Sorry, my fist day with Hugo.
Maybe anybody else does have an idea.

Maybe having a separate html for the faq´s is not the worst ... styling might become different from the expand.html later on.